File: Atividades/Entregavel1/Java/Main.java

/*
  ENTREGÁVEL 1 - AMBIENTES DE DESENVOLVIMENTO DE SOFTWARE
  Beatriz Vidal, 2217226, ADS
  Algoritmos fundamentais em Java
*/
import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    printIntro();
    Scanner sc1 = new Scanner(System.in);

    /*
     * ================= ALGORITMO 1 =================
     */
    System.out.println("1. Verificar se o número é primo");
    System.out.println("Digite um número: ");
    System.out.println(Primo.numeroPrimo(sc1.nextInt()));

    printLine();

    /*
     * ================= ALGORITMO 2 =================
     */
    System.out.println("2. Somar números");
    System.out.println("Digite a quantidade de números a serem somados: ");
    System.out.println(Soma.somarNumeros(sc1.nextInt()));

    printLine();

    /*
     * ================= ALGORITMO 3 =================
     */
    System.out.println("3. Série Fibonacci");
    System.out.println("Digite a quantidade de termos na Série Fibonacci: ");
    System.out.println(Fibonacci.fibonacci(sc1.nextInt()));

    printLine();

    /*
     * ================= ALGORITMO 4 =================
     */
    System.out.println("4. MDC");
    System.out.println("Digite o primeiro número: ");
    int mdc1 = sc1.nextInt();
    System.out.println("Digite o segundo número: ");
    int mdc2 = sc1.nextInt();
    System.out.println(Mdc.mdc(mdc1, mdc2));

    printLine();
    
    /*
     * ================= ALGORITMO 5 =================
     */
    Ordem.ordenarVetor();

    printLine();
    /*
     * ================= ALGORITMO 6 =================
     */
    System.out.println("6. Contagem");
    System.out.println("Digite o número final da contagem: ");
    Double maximo = sc1.nextDouble();
    System.out.println(Contagem.contagem(maximo));
  }

  /*
   * ============ FUNÇÕES AUXILIARES DE FORMATAÇÃO ============
   */
  public static void printLine() {
    System.out.println("\n------------------------------------------------------------\n");
  }

  public static void printIntro() {
    System.out.println("============================================================");
    System.out.println("Entregável 1 - Ambientes de Desenvolvimento de Software");
    System.out.println("Author: Beatriz Vidal");
    System.out.println("============================================================\n");
  }

}

File: Atividades/Entregavel1/Javascript/main.js
/*
  ENTREGÁVEL 1 - AMBIENTES DE DESENVOLVIMENTO DE SOFTWARE
  Beatriz Vidal, 2217226, ADS
  Algoritmos fundamentais em Javascript
*/
import { numeroPrimo } from "./primo.js";
import { somarNumeros } from "./soma.js";
import { mdc } from "./mdc.js";
import { fibonacci } from "./fibonacci.js";
import { ordenarVetor } from "./ordem.js";
import { contagem } from "./contagem.js";

console.log(`===============================
    ALGORITMOS FUNDAMENTAIS
    Author: Beatriz Vidal
===============================\n`)

/*
* ================= ALGORITMO 1 =================
*/
console.log("ALGORITMO 1: Número Primo");
console.log(numeroPrimo(5) + "\n");

/*
* ================= ALGORITMO 2 =================
*/
console.log("ALGORITMO 2: Somar números");
console.log(somarNumeros([5, 4, 7, 2]) + "\n");

/*
* ================= ALGORITMO 3 =================
*/
console.log("ALGORITMO 3: Série Fibonacci");
console.log(fibonacci(6) + "\n");

/*
* ================= ALGORITMO 4 =================
*/
console.log("ALGORITMO 4: MDC");
console.log(mdc(15, 13) + "\n");

/*
  * ================= ALGORITMO 5 =================
*/
console.log("ALGORITMO 5: Ordenar Vetor");
console.log(ordenarVetor([5, 7, 2, 1]) + "\n");

/*
  * ================= ALGORITMO 6 =================
*/
console.log("ALGORITMO 6: Contar Inteiros")
console.log(contagem(4) + "\n");